client/src/components/Draftail/blocks/MediaBlock.js

```diff
@@ -90,9 +90,13 @@ class MediaBlock extends Component {
         <span className="MediaBlock__icon-wrapper" aria-hidden>
           <Icon icon={entityType.icon} className="MediaBlock__icon" />
         </span>
-
         <img className="MediaBlock__img" src={src} alt={alt} width="256" />
 
+        {src ? null : (
```

Comment:
We’ll need to update the test case / add a new test case for this, checking that `fallbackText` does display.